common/ai_budget/00_dark_matter_budget.txt

```diff
@@ -0,0 +1,89 @@
+## Categories used in this file must have use_for_ai_budget = yes defined in economic_categories ##
```

Comment:
Since dark matter and some vanilla rare resources are still in the game I included them for simplicity's sake. But once we get our own custom resources fully put in I can easily make these files for those and remove the vanilla references